guide/message-components/buttons.md

```diff
@@ -0,0 +1,172 @@
+# Buttons
+
+The first type of interactive component we'll cover creating is the Button. Buttons are available in a variety of styles and can be used to provide permanent interfaces, temporary confirmation workflows, and other forms of additional interaction with your bot.
+
+::: tip
+This page is a follow-up to the [slash commands](/slash-commands/advanced-creation) section and [action rows](/message-components/action-rows) page. Please carefully read those pages first so that you can understand the methods used here.
+:::
+
+## Building buttons
+
+Buttons are one of the `MessageComponent` classes, which can be sent via messages or interaction responses.
+
+For this example, you're going to expand on the `ban` command that was previously covered on the [parsing options](/slash-commands/parsing-options.md) page with a confirmation workflow.
+
+To create your buttons, use the <DocsLink path="class/ButtonBuilder"/> class, defining at least the `customId`, `style` and `label`.
```

Comment:
I noticed the action row builder links to the new documentation, but this one doesn't. Is that intentional?